CONTRIBUTING.md

```diff
@@ -100,13 +100,28 @@ pytest
 pytest tests/${TARGET_TEST_FILE_NAME}
 ```
 
-## Continuous Integration
+## Continuous Integration and Local Verification
 
 Optuna repository uses GitHub Actions and CircleCI.
 
 Currently, we are migrating to GitHub Actions but still we use CirclCI for a test of `document`
 because it makes it much easier to check built documentation.
 
+### Local Verification
+
+By installing [`act`](https://github.com/nektos/act#installation) and Docker, you can run
+tests written for GitHub Actions locally.
+
+```bash
+JOB_NAME=checks
+act -j $JOB_NAME
```

Comment:
if it only supports `checks`, `act -j checks` is supposed to be simpler.

Reply:
As far as I know, `documentation` and `doctest` work as well.
`documentation` builds documentation with Sphinx on Python 3.8 and `doctest` runs doctest.

Also, without `-j checks`, i.e., only `act` runs `checks`, `documentation`, and `doctest`.

Reply:
In my env, I can not run `documentation` and `doctest` because virtual env does not have `make` command.

https://github.com/nektos/act#runners

In the above runners, I chosed `Medium Docker Image`. Maybe, that would be the reason.
Which runner did you use for act?

Reply:
I didn't know that. I've used large. So, it seems like the available jobs are up to users' choice of docker for act.